phantoms/MR_XCAT_qMRI/b_values.json

```diff
@@ -0,0 +1,7 @@
+[
+    [0.0, 1.0, 2.0, 5.0, 10.0, 20.0, 30.0, 50.0, 75.0, 100.0, 150.0, 250.0, 350.0, 400.0, 550.0, 700.0, 850.0, 1000.0],
```

Comment:
Rather than a list of lists perhaps an object so they can have names rather than just numbers?

Reply:
I tried object for the naming, but, we will have to enforce a pattern of naming on any one running the command for easy parsing. 

Something like { generic_<name> : [], ...}. That is why I made it into a list of list. But, if you think object is better approach then I can change it to that.

Reply:
I'd prefer objects. That way we have meaningful file names, like generic_short.json.

Some naming system would be good. I'm not sure what you mean "for easy parsing". I would imagine that people give either no name and they get all out or they give a name and they get that one out.

Reply:
This is still pending as well.

Reply:
Ok, I will change it to object.